src/main.rs

```diff
@@ -1,8 +1,17 @@
+extern crate env_logger;
+extern crate libc;
+#[macro_use]
+extern crate log;
+extern crate nix;
```

Comment:
should you add a cfg command for the `extern crate nix` as well ? as well as adding it in a target-specific part in Cargo.toml ? http://doc.crates.io/specifying-dependencies.html#platform-specific-dependencies